frontend/src/pages/LoginView.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Card } from '@components/ant-design/Card';
+import { Button } from '@components/ant-design/Button';
+import { Box } from '@components/atoms/Box';
+import { Paragraph, Title } from '@components/ant-design/Typography';
+import { LoginGraphic, Google } from '@components/svg';
+
+export const LoginView = () => {
+  const click = () => {
+    console.log('Button was clicked');
+  };
+
+  return (
+    <>
+      <Box style={{ textAlign: 'center', fontStyle: 'normal' }}>
+        <Card
+          pt={36}
+          pb={36}
+          width={606}
+          mt={88}
+          mb={49}
+          style={{ boxSizing: 'border-box', textAlign: 'center', marginLeft: 'auto', marginRight: 'auto' }}
```

Comment:
Jeśli istnieje konieczność dodania własnych styli do komponentu, to utwórz nowy ostylowany komponent z użyciem funkcji styled (jak przy tworzeniu komponentów w folderze `components`)